templates/mod/passenger.conf.erb

```diff
@@ -1,61 +1,262 @@
 # The Passenger Apache module configuration file is being
 # managed by Puppet and changes will be overwritten.
 <IfModule mod_passenger.c>
-  <%- if @passenger_root -%>
-  PassengerRoot "<%= @passenger_root %>"
+  <%- if @passenger_allow_encoded_slashes -%>
+  PassengerAllowEncodedSlashes <%= @passenger_allow_encoded_slashes %>
   <%- end -%>
-  <%- if @passenger_ruby -%>
-  PassengerRuby "<%= @passenger_ruby %>"
+  <%- if @passenger_app_env -%>
+  PassengerAppEnv <%= @passenger_app_env %>
+  <%- end -%>
+  <%- if @passenger_app_group_name -%>
+  PassengerAppGroupName <%= @passenger_app_group_name %>
+  <%- end -%>
+  <%- if @passenger_app_root -%>
+  PassengerAppRoot "<%= @passenger_app_root %>"
+  <%- end -%>
+  <%- if @passenger_app_type -%>
+  PassengerAppType <%= @passenger_app_type %>
+  <%- end -%>
+  <%- if @passenger_base_uri -%>
+  PassengerBaseURI <%= @passenger_base_uri %>
+  <%- end -%>
+  <%- if @passenger_buffer_response -%>
+  PassengerBufferResponse <%= @passenger_buffer_response %>
+  <%- end -%>
+  <%- if @passenger_buffer_upload -%>
+  PassengerBufferUpload <%= @passenger_buffer_upload %>
+  <%- end -%>
+  <%- if @passenger_concurrency_model -%>
+  PassengerConcurrencyModel <%= @passenger_concurrency_model %>
+  <%- end -%>
+  <%- if @passenger_data_buffer_dir -%>
+  PassengerDataBufferDir "<%= @passenger_data_buffer_dir %>"
+  <%- end -%>
+  <%- if @passenger_debug_log_file -%>
+  PassengerDebugLogFile <%= @passenger_debug_log_file %>
+  <%- end -%>
+  <%- if @passenger_debugger -%>
+  PassengerDebugger <%= @passenger_debugger %>
+  <%- end -%>
+  <%- if @passenger_default_group -%>
+  PassengerDefaultGroup <%= @passenger_default_group %>
   <%- end -%>
   <%- if @passenger_default_ruby -%>
   PassengerDefaultRuby "<%= @passenger_default_ruby %>"
   <%- end -%>
+  <%- if @passenger_default_user -%>
+  PassengerDefaultUser <%= @passenger_default_user %>
+  <%- end -%>
+  <%- if @passenger_disable_security_update_check -%>
+  PassengerDisableSecurityUpdateCheck <%= @passenger_disable_security_update_check %>
+  <%- end -%>
+  <%- if @passenger_enabled -%>
+  PassengerEnabled <%= @passenger_enabled %>
+  <%- end -%>
+  <%- if @passenger_error_override -%>
+  PassengerErrorOverride <%= @passenger_error_override %>
+  <%- end -%>
+  <%- if @passenger_file_descriptor_log_file -%>
+  PassengerFileDescriptorLogFile "<%= @passenger_file_descriptor_log_file %>"
```

Comment:
I don't quite understand why some values get quotes and some don't. Could you expand on this decision?